monkey/monkey_island/cc/ui/src/components/ui-components/InfoPane.js

```diff
@@ -42,11 +60,37 @@ function getSubtitle(props) {
 }
 
 function getBody(props) {
+  let body = [<span key={'body'}>{props.body}</span>];
+
+  if (props.warningType != WarningType.NONE) {
+    body.push(getWarning(props.warningType));
+  }
+
   return (
     <Card.Body className={'pane-body'}>
-      {props.body}
+      {body}
     </Card.Body>
   )
 }
 
-export default InfoPane
+function getWarning(warningType) {
+  if (warningType == WarningType.SINGLE) {
+    var warning = <span>This option may cause a system to become unstable or
+      may change a system's state in undesirable ways. Therefore, this option
+      is not recommended for use in production or other sensitive
+      environments.</span>;
+  } else {
+    warning = <span>Some options have been selected that may cause a system
+      to become unstable or may change a system's state in undesirable ways.
+      Running Infection Monkey in a production or other sensitive environment
+      with this configuration is not recommended.</span>;
```

Comment:
Use `Pluralize` here instead. 
```
Selected {Pluralize(option, warningCount)} may cause a system to become unstable or
      may change a system's state in undesirable ways. Therefore,  selected {Pluralize(option, warningCount)}
      is not recommended for use in production or other sensitive
      environments.</span>;
      ```

Reply:
`Pluralize` looks super cool, but I'm not sure it's the right solution here. Those messages are given in different contexts, I think the difference in the wordings makes the warnings more useful.